modules/compression/docs/README.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+<p class="badges">
+  <img src="https://img.shields.io/badge/From-v2.3-blue.svg?style=flat-square" alt="From-v2.3" /> 
+</p>
+
+The `@loaders.gl/decompress` module provides a small selection of lossless, legally unencumbered compression/decompression (aka deflate/inflate) "transforms" that work as plugins for loaders.gl
+
+## Compression Formats
+
+| Format                                        | Intended usage              | Notes                                                   |
+| --------------------------------------------- | --------------------------- | ------------------------------------------------------- |
+| [zlib](https://zlib.net/)                     | gzip(`.gz`) and Zip(`.zip`) | Essentially the `deflate' method in PKWARE's PKZIP 2.x) |
+| [lz4](https://github.com/lz4/lz4)             | Arrow Feather               | Optimized for speed (real-time compression)             |
+| [Zstandard](https://facebook.github.io/zstd/) | Arrow Feather               | Optimized for speed (real-time compression)             |
+
+## Decompression API
+
+The API offers "transforms" that can inflate/deflate data.
+
+| Transforms                                                                              | Sync | Description                      |
+| --------------------------------------------------------------------------------------- | ---- | -------------------------------- |
+| [`ZlibDeflateTransform`](modules/compression/docs/api-reference/zlib-deflate-transform) | Y    | Compress using Zlib codec        |
```

Comment:
Given that the loaders have a clean API with

```js
load(url, LoaderObject, loaderOptions);
```

It would seem clean to have a counterpart API for compression/decompression, something like:

```js
decompress(arrayBuffer, ZlibCompression, compressionOptions);
```

Reply:
Yes that is a reasonable idea. The proposed API will be more clear in the next PR which adds streaming, allowing these to be used to construct async iterators. Let's save more in depth discussions till then.